ptcg_cockatrice.xml

```diff
@@ -142,7 +142,7 @@
 			<pt>3/3</pt>
 			<tablerow>2</tablerow>
 			<text>B: Evolve from Ekans
-				When you evolve this Pokémon, it gets 2 +1/+1 counters and gains deathtouch.</text>
+				When this Pokémon evolves, put two +1/+1 counters on it and it gains deathtouch.</text>
```

Comment:
should these type of effects have `[cardname]` instead of "this Pokemon"?

Reply:
It's a mixed bag with existing cards. From what I can tell, [cardname] is used when the ability involves other creatures or when the creature is legendary. That's what I went with.

Reply:
I didn't actually read the change this was referencing because I was on mobile, but I think my comment still holds. I think I did "this Pokemon" for all evolve abilities, but I'm also pretty sure none of them reference other Pokemon so it should still work?